### PR TITLE
Add operator survey to repo

### PR DESCRIPTION
**Pull request by jberkus**
This adds the operator survey summary, in MD format, to this repo.  It also creates a WG subdirectory, which seems like a useful place to stash files related to WG activities.

### CODE REVIEW

**Comment by ahrtr** [comment]
Probably better to put the survey under the existing directory [docs](https://github.com/etcd-io/etcd-operator/tree/main/docs) (create a subdirectory `survey` under `docs`)?

File: wg/survey/2024-operator-survey.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Etcd Operator Survey Results
```
Comment (jmhbnz) [inline wg/survey/2024-operator-survey.md:1]: Nit: etcd is always lowercase.  There are several places in the doc to update this.
```suggestion
# etcd Operator Survey Results
```

**Comment by jberkus** [comment]
> Probably better to put the survey under the existing directory [docs](https://github.com/etcd-io/etcd-operator/tree/main/docs) (create a subdirectory `survey` under `docs`)?

Isn't the Docs folder going to eventually hold the docs for the actual operator?  In which case we don't want survey results there.

**Comment by jmhbnz** [comment]
> Isn't the Docs folder going to eventually hold the docs for the actual operator? In which case we don't want survey results there.

I would make a case for user facing operator docs to go on the etcd website for the most part.

**Comment by ahrtr** [comment]
> > Probably better to put the survey under the existing directory [docs](https://github.com/etcd-io/etcd-operator/tree/main/docs) (create a subdirectory `survey` under `docs`)?
> 
> Isn't the Docs folder going to eventually hold the docs for the actual operator? In which case we don't want survey results there.

The goal is to organize all documents under one directory. It's similar to [etcd documents](https://github.com/etcd-io/etcd/tree/main/Documentation), in which there are different kinds of docs.

**Comment by ahrtr** [comment]
@jberkus could you update this PR so that we can get it merged?

**Review by ahrtr — APPROVED** [review]
Let's merge this PR and I will raise a separate PR to resolve the comment. 

cc @jmhbnz @hakman

**Comment by ahrtr** [comment]
@hakman @jmhbnz Please approve this PR before I merge it. thx

**Pull request merged**